contrib/initramfs/scripts/zfs

```diff
@@ -228,7 +232,7 @@ import_pool()
 		"Importing pool '${pool}' using defaults"
 
 	ZFS_CMD="${ZPOOL} import -N ${ZPOOL_FORCE} ${ZPOOL_IMPORT_OPTS}"
-	ZFS_STDERR="$($ZFS_CMD "$pool" 2>&1)"
+	ZFS_STDERR="$(eval "${ZFS_CMD}" "$pool" 2>&1)"
```

Comment:
What does the `eval` do here? I'm not saying it's wrong, but I'm just not seeing what it does.

Reply:
Good catch, Richard! This was my foul attempt at fixing `IFS` issue that has been already fixed on master.